src/Symfony/Component/Asset/Tests/Context/RequestStackContextTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Asset\Tests\Context;
+
+use Symfony\Component\Asset\Context\RequestStackContext;
+
+class RequestStackContextTest extends \PHPUnit_Framework_TestCase
+{
+    public function testItIsAContext()
```

Comment:
IMO, this test brings no value. We don't have such tests in Symfony (and you don't do it for NullContext btw)